docs/releases/1.11.txt

```diff
@@ -492,6 +492,19 @@ This works similar to ``settings.TIME_ZONE = None`` except that it also sets
 if there's a use case where you find you can't adapt your code to set a
 ``TIME_ZONE``.
 
+
+``TransactionTestCase`` serialized data loading
+-----------------------------------------------
+
+Initial data migrations are now loaded in ``TransactionTestCase`` at the end of
+the test, after the database flush.
+In older versions, these data were loaded at the beginning of the test, but
```

Comment:
If this is meant to be a separate paragraph, it needs a blank line above it. Otherwise, please wrap it to the previous line.